### ISSUE TITLE
[BUG] Error computing memory usage of a categorical column

### ISSUE THREAD

**Issue by ayushdg** [rapidsai/cudf#4978]
**Describe the bug**
When trying to use the `memory_usage` method with a categorical column an errors is thrown

**Steps/Code to reproduce bug**
```python
import cudf
a = cudf.Series(['a', 'b', 'a']).astype("category")
a.memory_usage()
```
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-29-58b98e6d4ce3> in <module>
----> 1 a.memory_usage()

/conda/envs/rapids-14-april20-cu102/lib/python3.7/site-packages/cudf/core/series.py in memory_usage(self, index, deep)
    401
    402     def memory_usage(self, index=True, deep=False):
--> 403         n = self._column._memory_usage(deep=deep)
    404         if index:
    405             n += self._index.memory_usage(deep=deep)

/conda/envs/rapids-14-april20-cu102/lib/python3.7/site-packages/cudf/core/column/categorical.py in _memory_usage(self, deep)
    636         else:
    637             return (
--> 638                 self._categories._memory_usage()
    639                 + self.cat().codes.memory_usage()
    640             )

AttributeError: 'CategoricalColumn' object has no attribute '_categories'
```

**Environment overview (please complete the following information)**
  - 0.14 conda nightly from April20

**Comment by kkraus14** [comment]
@galipremsagar is this within scope of fixing in #4947?

**Comment by galipremsagar** [comment]
Yes, will fix it in #4947 as well.
